app/client/cypress/integration/national/national.spec.js

```diff
@@ -0,0 +1,189 @@
+describe('State Links', () => {
+  beforeEach(() => {
+    cy.visit('/national');
+  });
+
+  it('Clicking the “DISCLAIMER” button displays the disclaimer popup', () => {
+    // verify opening the disclaimer
+    cy.findByText('Disclaimer').click();
+    cy.findByText('EPA has posted information through this', {
+      exact: false,
+    }).should('exist');
+
+    // verify closing the disclaimer
+    cy.findByTitle('Close disclaimer').click();
+    cy.findByText('EPA has posted information through this', {
+      exact: false,
+    }).should('not.exist');
+  });
```

Comment:
@oftheheadland I know we're chatting on MS Teams, but you should take a look at a similar test Caleb wrote:

https://github.com/Eastern-Research-Group/mywaterway/blob/develop/app/client/cypress/integration/state/links.spec.js#L22

Probably best to be consistent if we can on:
1) prefer regex over string w/ `exact: false`
2) store regex/strings you're searching for in a variable